src/pages/AddressInfoPage/AddressTransactionRow.tsx

```diff
@@ -16,10 +16,11 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { explorer } from '@alephium/web3'
+import { isMempoolTx } from '@alephium/sdk'
+import { MempoolTransaction, Transaction } from '@alephium/web3/dist/src/api/api-explorer'
```

Comment:
Are you sure that this works well when building the app?

Reply:
I got no error when building.

Reply:
Hm, I would swear that there was a problem with importing like this and that's why I imported `explorer` and `node`...

Reply:
maybe something changed in the web3 package, very weird. But good news :)